### ISSUE TITLE
Navigation not working - After Enter Login

### ISSUE THREAD

**Issue by nunomsh** [driftyco/ionic-conference-app#397]
**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
After install and run the app, no changes to code, I notice if we enter LOGIN, but  do not hit the button to login, we can't go back to Map, Speakers, or About. I keeps in the login page

**Expected behavior:**
I should navigate to any page that does not requrire autentication

**Steps to reproduce:**
Just install the app

-->

**Other information:**
I've notice some warning in console: **main.js:78128 Didn't set nav root: invalid link: Tab**s 
This only apears AFTER enter the login

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
 Cordova CLI: 6.5.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.10.0
Xcode version: Not installed
```

Althought the Ionic and Cordova is not the latest version, I've tested with the latest cordova and ionic v3, same problem.
Install in device and same behavior as the browser

![ionic_cfa](https://cloud.githubusercontent.com/assets/2600175/25578911/f194db9e-2e6a-11e7-998a-3c35678e3e9d.gif)

**Comment by psamsotha** [comment]
Not sure what happened. When I submitted PR #392, I'm 100% sure the [`appPages` in the app.component](https://github.com/driftyco/ionic-conference-app/blob/master/src/app/app.component.ts#L44) used `TabsPage` as the `name`, that's why in the PR, the `name` was changed accordingly in the links to `TabsPage` (from the original `Tabs`). Now for some reason the `appPages` is using `Tabs` in the `app.component`. No the `name` in the `app.module` links is different.
